packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel.Plugin/src/Program.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Generator.CSharp.ClientModel
+{
+    internal static class Program
+    {
+        public static int Main(string[] args)
```

Comment:
customers should not need this is this left over or actually required?

Reply:
This seems to be required.